eng/versioning/version_client.txt

```diff
@@ -82,7 +82,7 @@ com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12;4.5.0-beta.1;4.5.0-beta.1
 com.azure:azure-cosmos-encryption;1.0.0-beta.9;1.0.0-beta.10
 com.azure:azure-data-appconfiguration;1.2.3;1.2.4
 com.azure:azure-data-appconfiguration-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-data-schemaregistry;1.0.0-beta.6;1.0.0-beta.7
+com.azure:azure-data-schemaregistry;1.0.0-beta.6;1.0.0
```

Comment:
Is there a reason why we just want to GA the client and not the SR apacheavro package? I thought customers would only interact with apacheavro package and don't need to use the client directly.

Reply:
There's been a lot of back and forth about how we want to structure the serializer... so we're not ready to GA the serializer but they can interact with the REST client.